ovos_audio/playback.py

```diff
@@ -20,14 +19,15 @@ def __init__(self, queue=TTS.queue, bus=None):
         self.queue = queue or TTS.queue
         self._terminated = False
         self._processing_queue = False
-        self._paused = False
+        self._do_playback = Event()
         self.enclosure = None
         self.p = None
         self._tts = []
         self.bus = bus or None
         self._now_playing = None
         self.active_tts = None
-        self._started = threading.Event()
+        self._started = Event()
+        # TODO: Config to disable
```

Comment:
why? plugins only load if explicitly enabled in config, no reason to put this behind a config. just dont enable plugins if you dont want to use this

Reply:
Not sure why I had that there.. The transformers service will take some extra time to init but this should be fine